llvm/lib/Target/RISCV/RISCVRegisterInfo.td

```diff
@@ -571,6 +571,16 @@ def GPRPF64 : RegisterClass<"RISCV", [f64], 64, (add
     X0_PD, X2_PD, X4_PD
 )>;
 
+let RegInfos = RegInfoByHwMode<[RV32, RV64], [RegInfo<32, 32, 32>, RegInfo<64, 64, 64>]> in
```

Comment:
I don't think these RegInfos are correct. I think they should be `RegInfo<64, 64, 64>, RegInfo<128, 128, 128>`.